web/styles/subscriptions.css

```diff
@@ -560,6 +571,24 @@ h4.user_group_setting_subsection_title {
     width: 100%;
 }
 
+.left .fa-ban,
+.right .fa-ban {
+    scale: 0.8;
+    opacity: 0.6;
+}
+
+.user-groups-list.hide-deactived-user-groups .deactivated-group {
+    display: none;
+}
+
+.user-groups-list.hide-active-user-groups .group-row {
+    display: none;
+}
+
+.user-groups-list:not(.hide-deactived-user-groups) .deactivated-group {
+    display: flex;
```

Comment:
This can be removed if you update the above selector as `.user-groups-list.hide-active-user-groups .group-row:not(.deactivated-group)`.